src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystem.java

```diff
@@ -364,6 +366,15 @@ private static void createImpedantBranch(LfBranch branch, LfBus bus1, LfBus bus2
             }
             sq1.addTerm(q1);
             branch.setQ1(q1);
+            // Add term slope * Q for svc
```

Comment:
Can you introduce an utility method that will be called here and just below? It will make the code more comprehensive.